README.md

```diff
@@ -55,8 +55,8 @@ Its options are described below.
 ```
   --verbose                       Show logs.
 
-  --after TEXT                    The date to begin analyzing the project at - has to be an iso date.
-                                  Example: 2019-10-11T19:10:38
+  --after-version TEXT            The version at which to start analyzing versions of the project.
```

Comment:
Could we update the parameters to use the same terminology as the code? So `--stop-version` rather than `--after-version` etc?

Reply:
Whoops, forgot to revert these changes to the README